ocs/constants.py

```diff
@@ -0,0 +1,32 @@
+"""
+Constants module.
+
+This module contains any values that are widely used across the framework,
+utilities, or tests that will predominantly remain unchanged.
+
+In the event values here have to be changed it should be under careful review
+and with consideration of the entire project.
+
+"""
+import os
+
+# Directories
+THIS_DIR = os.path.dirname(os.path.abspath(__file__))
+TOP_DIR = os.path.dirname(THIS_DIR)
```

Comment:
I'd argue that THIS_DIR and TOP_DIR shouldn't be stored anywhere

Reply:
`THIS_DIR` I understand, but that was originally just used to make constructing `TOP_DIR` a little more readable. Why wouldn't we want to keep `TOP_DIR` though?

Reply:
Oh, this is my mistake - blame it on a lack of coffee 🤦‍♂ 
Comment only stands for THIS_DIR

Reply:
Totally agree, will remove `THIS_DIR`